addons/l10n_my_edi/models/account_move.py

```diff
@@ -493,8 +503,10 @@ def _cron_l10n_my_edi_synchronize_myinvois(self):
                 # Commit if we can, in case an issue arises later.
                 if self._can_commit():
                     self._cr.commit()
+                time.sleep(1)  # We don't want to send api calls nonstop, so we add some delay.
```

Comment:
Isn't 1s a bit much?

Reply:
This is arbitrary, so I'm open to reducing it. 1 is quite a lot yes